src/components/Labelling/LabellingFlow.js

```diff
@@ -192,7 +191,7 @@ export default class LabellingFlow extends Component {
   }
 
   descriptionCancel = () => {
-    const { description = '' } = cloneDeep(this.state)
+    const { description = '' } = JSON.parse(JSON.stringify(this.state))
```

Comment:
I do not think this is a good solution for deep clone since it converts some data types like Date to string, so it is really not the exact clone.

Reply:
You're right, Moving back to cloneDeep